walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css

```diff
@@ -21,29 +21,29 @@
   text-align: center;
 }
 
+.cell {
+  background: #E0FFFF;
+  border-style: solid;
+  color: black;
+  text-align: center;
+}
+
 .color {
   color: blueviolet;
 }
 
+.coloring {
```

Comment:
Nit: I'm not sure if `coloring` is the most explanatory name for what this class does style-wise

Reply:
You are right, will fix now!